lib/log.js

```diff
@@ -66,17 +66,22 @@ Lug.prototype.event = function (name, data) {
   process.stdout.write(JSON.stringify(e) + '\n')
 }
 
-Lug.prototype.activityEvent = function(event, uid, request) {
+Lug.prototype.activityEvent = function(event, request, data) {
   var info = {
-    event: event,
-    uid: uid,
```

Comment:
One thing that's now less obvious, is that `data` *must* have a `uid` property in order for the logs to be useful.  I wonder if we should explicitly check for this and throw, or just depend on tests to enforce it.